test/data/test_utils.py

```diff
@@ -45,3 +46,19 @@ def test_get_tokenizer_toktokt(self):
             data.get_tokenizer(1)
         with self.assertRaises(ValueError):
             data.get_tokenizer("some other string")
+
+    def test_text_nomalize_function(self):
+        # Test text_nomalize function in torchtext.datasets.text_classification
+        ref_lines = []
+        test_lines = []
+        with open('../asset/test.csv') as src_data:
+            for line in src_data:
+                test_lines.append(text_normalize(line))
+
+        with open('../asset/ag_news.test') as ref_data:
```

Comment:
Please use the csv reader abstraction for this, it'll be more robust and aligned with the rest of the code

```
    from torchtext.utils import unicode_csv_reader

    with io.open(data_path, encoding="utf8") as f:
        reader = unicode_csv_reader(f)
        for row in reader:
```